models/articles.model.js

```diff
@@ -13,3 +13,24 @@ exports.fetchArticleById = (articleId) => {
       return response.rows[0];
     });
 };
+
+exports.updateArticleById = (articleId, increaseVotesBy) => {
+  if (!increaseVotesBy) {
+    return Promise.reject({ status: 400, message: "Bad request" });
+  }
+
```

Comment:
What if increaseVotesBy is an empty { } or {rating: 50} . Error message could be more specific like "missing required fields"